src/history/__tests__/historyReducer.test.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  CHANGE_DATE_FILTER_ACTION,
+  CHANGE_PRICE_FILTER_ACTION,
+  ChangeDateFilterAction,
+  ChangePriceFilterAction,
+  HISTORY_RECEIVED_ACTION,
+  HistoryReceivedAction
+} from 'history/historyActions'
+import historyReducer from 'history/historyReducer'
+
+import { mockChartData, mockEmptyHistoryState, mockHistoryState } from './__mock__/mockData'
+
+describe('history reducer', () => {
+  it('should return the initial state', () => {
+    expect(historyReducer(undefined, {} as HistoryReceivedAction)).toEqual(mockEmptyHistoryState)
```

Comment:
what is the difference between `initialState` and `mockEmptyHistoryState`. Which one should be tested here? What will happen when you add a new field to the history state?